### PR TITLE
Add qa-orchestra QA lifecycle agents

### PR DESCRIPTION
**Pull request by Anasss**
## Summary

Adds 5 QA lifecycle agents from [qa-orchestra](https://github.com/Anasss/qa-orchestra), a coordinated 10-agent QA toolkit for Claude Code:

- **functional-reviewer** — Compares code diffs against acceptance criteria to find gaps
- **test-scenario-designer** — Generates test scenarios from AC (happy path, negative, boundary, edge cases)
- **bug-reporter** — Converts QA findings into structured, developer-ready bug reports
- **automation-writer** — Turns test scenarios into executable Playwright/Cypress test code
- **qa-orchestrator** — Routes QA tickets to the right agents in the right order

These agents work together via output chaining — each agent writes to a file that the next agent reads. The orchestrator coordinates the full pipeline.

See the full 10-agent set (including environment-manager, manual-validator, and browser validation) at https://github.com/Anasss/qa-orchestra

### CODE REVIEW

File: content/04-quality-security/qa-orchestrator.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: qa-orchestrator
+description: Routes QA tickets to specialized agents in the right order — orchestrates functional review, test scenario design, automation writing, bug reporting, and browser validation across the full QA lifecycle. Part of the qa-orchestra toolkit.
```
[the remark this answers is not in the document]
Comment (Anasss) [reply content/04-quality-security/qa-orchestrator.md]: Resoved!

File: content/04-quality-security/qa-orchestrator.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Request routing:
+- "Review this ticket" -> functional-reviewer, then bug-reporter (if gaps)
+- "Create test cases" -> test-scenario-designer
+- "Automate these scenarios" -> automation-writer
+- "Test this PR" -> functional-reviewer + test-scenario-designer (parallel) -> bug-reporter + automation-writer
+- "Full QA pipeline" -> all agents in sequence with parallelism
+
+Parallelism rules:
+- functional-reviewer and test-scenario-designer can run in parallel (both only need AC)
```
[the remark this answers is not in the document]
Comment (Anasss) [reply content/04-quality-security/qa-orchestrator.md]: Fixed!

**Comment by ayush-that** [comment]
Thanks for the PR @Anasss!
